packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -1,205 +1,156 @@
+import React, { useState } from 'react'
+import { useTranslation } from 'react-i18next'
 import { ScrollView, StyleSheet, Text, View } from 'react-native'
-import Modal from 'react-native-modal'
-import SafeAreaView from 'react-native-safe-area-view'
-import { connect } from 'react-redux'
+import { useSafeArea } from 'react-native-safe-area-view'
+import { useDispatch } from 'react-redux'
 import { ErrorMessages } from 'src/app/ErrorMessages'
 import ErrorMessageInline from 'src/components/ErrorMessageInline'
-import { Namespaces, withTranslation } from 'src/i18n'
+import i18n, { Namespaces } from 'src/i18n'
 import { setHasSeenVerificationNux } from 'src/identity/actions'
 import { isUserBalanceSufficient } from 'src/identity/utils'
 import { INVITE_FEE } from 'src/invite/saga'
-import { nuxNavigationOptions } from 'src/navigator/Headers'
-import { navigate, navigateHome } from 'src/navigator/NavigationService'
+import { HeaderTitleWithSubtitle, nuxNavigationOptions } from 'src/navigator/Headers.v2'
+import { navigateHome } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { RootState } from 'src/redux/reducers'
+import { TopBarTextButton } from 'src/navigator/TopBarButton.v2'
+import { StackParamList } from 'src/navigator/types'
+import useSelector from 'src/redux/useSelector'
+import VerificationLearnMoreDialog from 'src/verify/VerificationLearnMoreDialog'
+import VerificationSkipDialog from 'src/verify/VerificationSkipDialog'
 
 const VERIFICATION_FEE_ESTIMATE = Number(INVITE_FEE) * 0.9
 
-interface DispatchProps {
-  setHasSeenVerificationNux: typeof setHasSeenVerificationNux
-}
-
-interface StateProps {
-  userBalance: string | null
-}
+type ScreenProps = StackScreenProps<StackParamList, Screens.VerificationEducationScreen>
 
-type Props = WithTranslation & DispatchProps & StateProps
+type Props = ScreenProps
 
-interface State {
-  isModalVisible: boolean
-}
+function VerificationEducationScreen({ route, navigation }: Props) {
+  const showSkipDialog = route.params?.showSkipDialog || false
+  const [showLearnMoreDialog, setShowLearnMoreDialog] = useState(false)
+  const { t } = useTranslation(Namespaces.onboarding)
+  const userBalance = useSelector((state) => state.stableToken.balance)
+  const balanceIsSufficient = isUserBalanceSufficient(userBalance, VERIFICATION_FEE_ESTIMATE)
+  const dispatch = useDispatch()
+  const headerHeight = useHeaderHeight()
+  const insets = useSafeArea()
 
-const mapStateToProps = (state: RootState): StateProps => ({
-  userBalance: state.stableToken.balance,
-})
-
-const mapDispatchToProps = {
-  setHasSeenVerificationNux,
-}
-
-class VerificationEducationScreen extends React.Component<Props, State> {
-  static navigationOptions = nuxNavigationOptions
-
-  state: State = {
-    isModalVisible: false,
-  }
-
-  onPressLearnMore = () => {
-    navigate(Screens.VerificationLearnMoreScreen)
+  function onPressStart() {
```

Comment:
similar question on fat arrows vs function keyword - any preference?

Reply:
no strong preference either, the `function` keyword makes it arguably slightly more readable, but fat arrow makes it easy to wrap with `useCallback` for instance 😄